docs/content/querying/filters.md

```diff
@@ -58,18 +58,28 @@ The filter specified at field can be any other filter defined on this page.
 
 ### JavaScript filter
 
-The JavaScript filter matches a dimension against the specified JavaScript function predicate. The filter matches values for which the function returns true.
+The JavaScript filter matches dimensions against the specified JavaScript function predicate. The filter matches values for which the function returns true.
 
-The function takes a single argument, the dimension value, and returns either true or false.
+The function takes same number of arguments as the dimension values, and returns either true or false.
 
 ```json
 {
   "type" : "javascript",
-  "dimension" : <dimension_string>,
-  "function" : "function(value) { <...> }"
+  "dimension" : <a dimension_string>,
+  "dimensions" : <array of dimension_strings>,
+  "function" : "function(value1, value2, ...) { <...> }"
+  "rowBased" : "false"
 }
 ```
 
+|property|description|required?|
+|--------|-----------|---------|
+|type|This String should always be "javascript".|yes|
+|dimension|The dimension to perform the search over.|either of dimension or dimensions|
+|dimensions|The dimensions to perform the search over.|either of dimension or dimensions|
+|function|A JSON function to be applied.|yes|
+|rowBased|If this is true, multi-valued column is handed over as single object array to function. In this case, bitmap index cannot be applied. Default: false(insensitive)|no|
```

Comment:
@navis ok, I'm asking because the documentation doesn't explain what happens in that case. Given that this is a pretty hard to understand flag, it's important we explain in detail what happens.